frontend/src/components/ThankyouForApplying.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from "./ThankyouForApplying.module.css";
+
+import { Page } from "../components/index.ts";
```

Comment:
address this. It should be fixed using `npm run lint-fix` in the frontend directory